src/osd/OSD.cc

```diff
@@ -1660,6 +1660,13 @@ bool OSD::asok_command(string command, cmdmap_t& cmdmap, string format,
     } else {
       op_tracker.dump_ops_in_flight(f);
     }
+  } else if (command == "dump_blocked_ops") {
+    if (!op_tracker.tracking_enabled) {
+      ss << "op_tracker tracking is not enabled";
```

Comment:
This concerns me. Imagine:
*) user has tracking disabled
*) Op gets stuck and user notices the cluster is blocked
*) user runs dump_blocked_ops and gets this message
*) user enables tracking
*) user runs dump_blocked_ops, sees that none are listed, and assumes the error is elsewhere

This discussion probably belongs on #5168, but I think we need some way of indicating that there are untracked ops which might be blocked, if the system has ever had the op tracking disabled. I'm not sure how to do that; thoughts? :/